### ISSUE TITLE
e2e test: could not start etcd process cluster

### ISSUE THREAD

**Issue by gyuho** [coreos/etcd#4511]
```
=== RUN   TestCtlV2WatchWithProxy
--- FAIL: TestCtlV2WatchWithProxy (0.02s)
	etcdctl_test.go:166: could not start etcd process cluster (couldn't get expected output: 'etcdserver: set the initial cluster version to')
```

https://semaphoreci.com/coreos/etcd/branches/pull-request-4507/builds/4

**Comment by heyitsanthony** [comment]
May have fixed this one in 1719bc0b0c984c89188837aedf1ec08f55983676. It appeared `child.Capture()` was racing to eat the ready string so Expect wasn't working as intended.

**Comment by gyuho** [comment]
@heyitsanthony I see. Ok let's close this for now. Thanks!
